extensions/ext/GLSL_EXT_ray_flags_primitive_culling.txt

```diff
@@ -73,6 +74,40 @@ Modifications to the OpenGL Shading Language Specification, Version 4.60
       #define GL_EXT_ray_flags_primtive_culling          1
 
 
+Additions to Chapter 4 of the OpenGL Shading Language Specification
+(Variables and Types)
+
+    Modify Section 4.4, Layout Qualifiers
+
+    Modify the sentence that begins with "They can also appear with
+    just an interface-qualifier" to add "or even without one"
+
+    (Add additional line)
+    layout-qualifier;
+
+
+    Modify table in Section 4.4 (Layout Qualifiers)
+
+
+    Layout Qualifier    Qualifier  Individual   Block    Block      Allowed
+                          Only      Variable             Member    Interface
+    -----------------   ---------  ----------   ------   ------    ---------
+
+    primitive_culling      X                                         None
+
+
+    Add a new subsection to Section 4.4
+
+    4.4.10 Primitive Culling Layout Qualifier
+
+    This layout qualifier can only be applied at global scope
+    and without an interface-qualifier.
+
+    It is a compile-time error to use this qualifier with an
+    interface-qualifier, on an interface block or member of an interface
+    block, or with any other layout qualifiers.
+
+
```

Comment:
should it be a compile time error if either of the new constants are used without having this declared?

Reply:
Cannot really prevent someone from using values '256U' anyway, so erroring out for one case doesnt make sense . All this qualifier does is enables a capability which defines valid behavior for the flag values? Otherwise results are undefined

Reply:
fair enough.